src/chainstate/stacks/boot/mod.rs

```diff
@@ -68,17 +74,26 @@ lazy_static! {
         format!("{}\n{}", BOOT_CODE_POX_MAINNET_CONSTS, BOOT_CODE_POX_BODY);
     pub static ref BOOT_CODE_POX_TESTNET: String =
         format!("{}\n{}", BOOT_CODE_POX_TESTNET_CONSTS, BOOT_CODE_POX_BODY);
+    // TODO (#3034) - replace when exit contract is published
+    static ref BOOT_CODE_EXIT_AT_RC_MAINNET: String =
+        format!("{}\n{}", BOOT_CODE_EXIT_AT_RC_MAINNET_CONSTS, BOOT_CODE_EXIT_AT_RC);
+    pub static ref BOOT_CODE_EXIT_AT_RC_TESTNET: String =
+        format!("{}\n{}", BOOT_CODE_EXIT_AT_RC_TESTNET_CONSTS, BOOT_CODE_EXIT_AT_RC);
+
     pub static ref BOOT_CODE_COST_VOTING_TESTNET: String = make_testnet_cost_voting();
-    pub static ref STACKS_BOOT_CODE_MAINNET: [(&'static str, &'static str); 6] = [
+    // TODO (#3034): remove exit-at-rc once deployed
```

Comment:
I think that, because even once `exit-at-rc` is deployed we'll need to keep integration tests around that reference an instantiated contract, it may be better to pay the implementation cost of making this more configurable now. I'd suggest something like the following:

1. Make the `QualifiedContractIdentifier` corresponding to the `exit-at-rc` contract configurable via the `exit_at_constants` struct. The code in the coordinator can use that identifier to perform its checks (it will need to check for the existence of the contract).
2. For the "unit tests" (i.e., the tests within the `/src/` directory), I think it'll be easy to instantiate the contract synthetically (basically, the way it's done now, but without including the contract specifically in the `STACKS_BOOT_CODE_TESTNET` array).
3. For the "integration tests" (i.e., the ones in `neon_integrations.rs`), it should be possible to instantiate the contract using a normal contract publish transaction.